doghouse/service.go

```diff
@@ -0,0 +1,31 @@
+package doghouse
+
+// GitHub Check Request
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported type CheckRequest should be of the form "CheckRequest ..." (with optional leading article)